packages/constants/src/chain.ts

```diff
@@ -1,14 +1,20 @@
 import { NETWORK } from './network';
 
+export enum MAINNET_CHAINS {
+  COLUMBUS4 = 'columbus-4',
```

Comment:
do we really need anything columbus-4 related?

Reply:
removed 👍